garrysmod/lua/includes/extensions/util.lua

```diff
@@ -355,7 +355,9 @@ end
 local function DistanceToTrace( pos, traceStart, traceEnd, along, pointOnTrace )
 	local to = pos - traceStart
 	local dir = traceEnd - traceStart
-	local length = dir:Normalize()
+	
+	dir:Normalize()
+	local length = dir

```

Comment:
I don't believe so as the C++ code does "float length = dir.NormalizeInPlace();"